phoenicis-repository/src/main/java/org/phoenicis/repository/repositoryTypes/ClasspathRepository.java

```diff
@@ -76,6 +79,11 @@ public RepositoryDTO fetchInstallableApplications() {
         }
     }
 
+    @Override
+    public boolean isSafe() {
```

Comment:
I see. In my opinion we should rather rewrite that part. I wouldn't like to have it in the classpath (e.g. you cannot fork that). Moreover, Wine serves as an example implementation for any other engine someone might want to implement.